### ISSUE TITLE
So many pre-installed packages for a Base image

### ISSUE THREAD

**Issue by joeky888** [manjaro/manjaro-docker#16]
Image size is a problem that keeps users stay away from it. Could we have another minimal Manjaro docker image that only installs the basics? Pulling the latest amd64 image requires 364MB bandwidth (1.08GB unpacked). We probably need a rootless image that an AUR-helper like Yay is pre-installed but that's for another issue.

One could compare the list of packages from different implementations using these commands:

```fish
# Window a
docker run --rm -it --entrypoint bash archlinux
pacman -Qe
# Window b
docker run --rm -it --entrypoint bash manjarolinux/base
pacman -Qe
```

I have got some ideas for a minimal image:

TL;DR :+1:  `pacman -Rsuc systemd git git-lfs cmake make automake binutils gettext m4 pkgconf sudo texinfo libtool gcc gawk bison groff patch which gzip grep libxml2 elfutils`

1. Remove Systemd, Docker itself already takes care of what Systemd does.
2. Remove Git.
3. Remove base-devel (Not entirely, Pacman is still vital) and Cmake

**Comment by boredland** [comment]
Hi, fair enough. The image currently has been mainly in use as a "base" for running builders inside, so some dependencies were needed there on a regular basis. But you're right, it may be exhaustive and could be easily solves by having multiple images.

Just out of curiosity: what are you planning to do with a docker Linux image?

**Comment by ifurther** [comment]
Maybe can split two container base and base-devel, it can provide just use simple software

**Comment by boredland** [comment]
Sounds like a plan. How should we name them? "base" and "base-devel"?

**Comment by boredland** [comment]
Soo, I think the setup works. We may need to change scopes of the token we use for the dockerhub pushes.

Base is now 240MB on amd64, which looks more like it.

Open questions:
- why is the arm64 image so huge?
- how to remove systemd? its part of the base group and things depend on it...

**Comment by joeky888** [comment]
> how to remove systemd?

Looks like it is impossible, but perhaps we could also remove `pacman-mirrors` and the whole python dependencies for base.

**Comment by boredland** [comment]
found the issue with the giant arm image:

![image](https://user-images.githubusercontent.com/4662748/187708360-361dfeda-12ac-4154-99a9-ba167310fd45.png)

**Comment by joeky888** [comment]
Nice! These changes are indeed magnificent. Thank you!
